external-import/recorded-future/src/rflib/rf_alerts.py

```diff
@@ -312,18 +312,33 @@ def alert_to_incident(self, alert):
                     bundle_objects.append(stix_relationship)
 
                 elif entity["type"] == "IpAddress":
-                    stix_ipv4address = stix2.IPv4Address(
-                        value=entity["name"],
-                        object_marking_refs=self.tlp,
-                        custom_properties={
-                            "x_opencti_created_by_ref": self.author["id"],
-                        },
-                    )
-                    stix_relationship = self._generate_stix_relationship(
-                        stix_ipv4address.id, "related-to", stix_incident.id
-                    )
-                    bundle_objects.append(stix_ipv4address)
-                    bundle_objects.append(stix_relationship)
+                    stix_ip_address = None
+                    ip_address_value = entity["name"]
+                    if is_ip_v4_address(ip_address_value):
```

Comment:
```suggestion
                    if validators.ipv4(ip_adress_value):
```
Doing prevent you from creating the functions bellow (same for ipv6)

Reply:
I chose not to use `validators` lib as it wasn't listed as a dependency in `requirements.txt`, that's why I'd rather use python's core module.